src/component-node.ts

```diff
@@ -24,12 +27,18 @@ export default class ComponentNode {
     }
   }
 
-  public addDependency(componentName: string) {
+  public addComponentDependency(componentName: string) {
+    if (!this.dependencyCategories.includes('HTTP')) {
+      this.dependencyCategories.push('HTTP')
+    }
```

Comment:
keep flip flopping over this - it will now add a component category of `HTTP` to anything that relies on another service. 
Might be a bit superfluous but is a bit more consistent with when a service relies on another http resource that we haven't identified.